napari/layers/shapes/shapes.py

```diff
@@ -2939,7 +2939,7 @@ def to_labels(self, labels_shape=None):
             For overlapping shapes z-ordering will be respected.
         """
         if labels_shape is None:
-            labels_shape = self._extent_data[1] - self._extent_data[0]
+            labels_shape = self._extent_data[1] + 1
```

Comment:
Maybe add a comment here @jni on why the 1, since it could have been a 0.5 also :) ?